components/splitter/accessibility/overview.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Accessibility Overview
+page_title: Telerik UI for Blazor Splitter Documentation | Splitter Accessibility Overview
+description: "Get started with the Telerik UI for Blazor Splitter and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.2."
+tags: telerik,blazor,accessibility,wai-aria,wcag,splitter
+slug: splitter-accessibility-overview
+position: 0
+---
+
+# Accessibility Overview
+
+The UI for Blazor Splitter component is [WCAG 2.2 AA](https://www.w3.org/TR/WCAG22) and [Section 508](https://www.section508.gov) compliant. The component also follows the [WAI-ARIA best practices](https://www.w3.org/WAI/ARIA/apg/) for implementing the keyboard navigation for its component [role](https://www.w3.org/TR/wai-aria/#roles), and is tested against the popular screen readers.
```

Comment:
Ideally, links leading to resources outside of the Blazor doc site need to use the `<a href="" target="_blank">` syntax. See [here ](https://docs.telerik.com/style-guide/cross-references#guidelines) for reference.